packages/plugin-pnp/sources/index.ts

```diff
@@ -90,6 +93,11 @@ const plugin: Plugin<CoreHooks & StageHooks> = {
       type: SettingsType.STRING,
       default: `pnp`,
     },
+    minizip: {
+      description: `Whether Yarn should use minizip to extract archives`,
+      type: SettingsType.BOOLEAN,
+      default: false,
+    },
```

Comment:
Since the main benefit of this PR is to decrease the runtime footprint (by comparison the install doesn't have a lot to gain, since only unplugged packages are extracted on the disk and there's very little of them), I'd rather keep the existing codepath for unplug (to simplify the changes as much as possible).